website/pages/en/sunrise.mdx

```diff
@@ -82,7 +84,27 @@ Around the start of June, Edge & Node will automatically upgrade actively querie
 
 ### What happens if I don't upgrade my subgraph?
 
-Subgraphs will be queryable on the hosted service until June 12th. After this date, the hosted service homepage will still be accessible, however, query endpoints will no longer be available. Owners of hosted service subgraphs will still be able to upgrade their subgraphs to The Graph Network after June 12th, though earlier upgrades are entitled to [earn rewards](https://thegraph.com/sunrise-upgrade-program/). Developers will also be able to claim [auto-upgraded subgraphs](https://thegraph.com/blog/unveiling-updated-sunrise-decentralized-data/#phase-2-sunbeam)
+Subgraphs will be queryable on the hosted service until June 12th. After this date, the hosted service homepage will still be accessible, however, query endpoints will no longer be available. Owners of hosted service subgraphs will still be able to upgrade their subgraphs to The Graph Network after June 12th, though earlier upgrades are entitled to [earn rewards](https://thegraph.com/sunrise-upgrade-program/). Developers will also be able to claim [auto-upgraded subgraphs](https://thegraph.com/blog/unveiling-updated-sunrise-decentralized-data/#phase-2-sunbeam).
+
+### What should I do with my old subgraphs? Will they stop working and should I delete them?
+
+It is not possible to delete subgraphs, but they can be deprecated. To learn how to deprecate a subgraph, check out [deprecating a subgraphs](https://thegraph.com/docs/en/managing/deprecating-a-subgraph/). Query endpoints will no longer be available, and owners will not be able to deploy new subgraphs to the hosted service.
+
+### Will Subgraph Studio be impacted by this upgrade?
+
+No, Subgraph Studio will not be impacted by Sunrise.
+
+### Will the hosted service be closed?
```

Comment:
Not sure what "closed" means here - maybe "What will happen to the hosted service?"

Reply:
```suggestion
### What will happen to the hosted service?
```